pkg/controller/statusmanager/pod_status.go

```diff
@@ -251,6 +252,16 @@ func (status *StatusManager) SetFromPods() {
 		}
 	}
 
+	// hack for 2-phase upgrade from non-IC to IC ovnk:
+	// don't update the version field until phase 2 is over
+	if icConfigMap, err := util.GetInterConnectConfigMap(status.client.ClientFor("").Kubernetes()); err == nil {
+		// When an upgrade from <= 4.13 is ongoing, the IC configmap exists and exhibits ongoing-upgrade=true.
+		// When multizone control-plane and node have been rolled out (end of phase 2), the configmap is deleted.
+		if ongoingUpgrade, ok := icConfigMap.Data["ongoing-upgrade"]; ok && ongoingUpgrade == "true" {
+			reachedAvailableLevel = false
+		}
+	}
```

Comment:
It'll retry the next time status manager gets triggered. Would you something else? if it fails to contact the api server, I guess it can only try again later on

Reply:
@ricky-rav did you decide to not act on this comment or missed it?

Reply:
so I'm printing the error now, which is indeed useful. The entire function doesn't really return when an error occurs, it just prints it, so I kept the same behaviour...

Reply:
True but the other errors were not really preventing us to assess the overall state of the operator, they were just errors that we had when reporting some details of that state.

But this error is a bit different because it doesn't allow us to asses the state, we might be incomplete because a zone migration is progressing but we can miss that and report as complete.

So in addition to printing the error, should we keep `reachedAvailableLevel` set to `false`?

Reply:
Oh ok... I see now. If that's fine with everyone, it can be one of the first refinements after merging this PR. The fix is just this: https://github.com/openshift/cluster-network-operator/pull/1939/commits/8fbe0ad1f752351bd195ba6f1ce4427d2db6bc9c